docs/features/search/how-to-guides.md

```diff
@@ -386,14 +386,27 @@ import { searchModuleTechDocsCollator } from '@backstage/plugin-search-backend-m
 import { searchModuleExploreCollator } from '@backstage/plugin-search-backend-module-explore/alpha';
 
 const backend = createBackend();
-// adding the search plugin to the backend
+// [1] adding the search plugin to the backend
 backend.add(searchPlugin());
-// (optional) the default search engine is Lunr, if you want to extend the search backend with another search engine.
+// [2] (optional) the default search engine is Lunr, if you want to extend the search backend with another search engine.
 backend.add(searchModuleElasticsearchEngine());
-// extending search with collator modules to start index documents, take in optional schedule parameters.
+// [3] extending search with collator modules to start index documents, take in optional schedule parameters.
 backend.add(searchModuleCatalogCollator());
 backend.add(searchModuleTechDocsCollator());
 backend.add(searchModuleExploreCollator());
 
 backend.start();
 ```
+
+To create your own collators/decorators modules, please use the [searchModuleCatalogCollator](https://github.com/backstage/backstage/blob/d7f955f300893f50c4882ea8f5c09aa42dfaacfd/plugins/search-backend-module-catalog/src/alpha.ts#L49) as an example, we recommend that modules have their own packages. You can also find the the available search engines and indexers modules documentation in the Alpha API reports:
+
+**Search engines**
+
+- Postgres [module](https://github.com/backstage/backstage/blob/d7f955f300893f50c4882ea8f5c09aa42dfaacfd/plugins/search-backend-module-pg/alpha-api-report.md);
+- Elasticsearch [module](https://github.com/backstage/backstage/blob/d7f955f300893f50c4882ea8f5c09aa42dfaacfd/plugins/search-backend-module-elasticsearch/alpha-api-report.md).
+
+**Search indexers**
```

Comment:
Maybe we can use "Search collators/decorators" to follow our same terminology as explained in the concepts documentation?

Reply:
Done!